.github/workflows/main.yml

```diff
@@ -14,11 +14,10 @@ jobs:
     - uses: actions/checkout@v2
     - run: npm install
     - run: npx aegir lint
-    - uses: gozala/typescript-error-reporter-action@v1.0.8
```

Comment:
What the reason for this change ? I personally find inline comments a lot more convenient than reading CI output